### PR TITLE
Make logging optional

### PR DESCRIPTION
**Pull request by smartdanny**

### CODE REVIEW

File: fuse/dl/lightning/pl_module.py [hunk 1]
```diff
@@ -82,8 +82,12 @@ def __init__(
         """
         super().__init__(**kwargs)
 
+        if (save_arguments or save_model) and (model_dir is None):
```
Comment (mosheraboh) [inline fuse/dl/lightning/pl_module.py:85]: Are you avoiding from lightning logging? or maybe it's configurable in trainer?

File: fuse/dl/lightning/pl_module.py [hunk 2]
```diff
@@ -32,8 +32,8 @@ class LightningModuleDefault(pl.LightningModule):
 
     def __init__(
         self,
-        model_dir: str,
         model: torch.nn.Module,
+        model_dir: str = None,
```
[the remark this answers is not in the document]
Comment (mosheraboh) [reply fuse/dl/lightning/pl_module.py:36]: Please also update the comments and change the type to Optional[str]

**Review by mosheraboh — CHANGES_REQUESTED** [review]
Thanks @smartdanny !
See my comments inline and let me know if don't agree with any of them.

File: fuse/dl/lightning/pl_module.py (hunk 1 above)
Comment (smartdanny) [reply fuse/dl/lightning/pl_module.py:85]: Yes, users can either only use lightning logging, or not log at all (which can be useful for debugging).

**Review by mosheraboh — APPROVED** [review]
LGTM

**Pull request merged**